src/ishikawa_backend.erl

```diff
@@ -45,10 +45,6 @@
 
 -define(PEER_SERVICE, partisan_peer_service).
 
--record(state, {actor :: actor(),
```

Comment:
Since each module may have it's own state, the state should stay here and not be in an include file.